### PR TITLE
[WIP] removed cartodb.com references and added on premise version

### PR DESCRIPTION
**Pull request by javisantana**
(waiting to merge with master after @xavijam updates)

- removes cartodb.com references
- removes "upgrade" stuff
- adds onpremise version to the footer

cc @santisaez @xavijam (review it please)

### CODE REVIEW

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/3572/))

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -70,7 +70,7 @@ def frontend_config
       maps_api_template:          maps_api_template,
       sql_api_template:           sql_api_template,
       user_name:                  CartoDB.extract_subdomain(request),
-      cartodb_com_hosted:         Cartodb.config[:cartodb_com_hosted],
+      cartodb_com_hosted:         Cartodb.config[:cartodb_com_hosted].present?,
```
Comment (xavijam) [inline app/helpers/application_helper.rb]: There is a method in application_helper.rb called ```cartodb_com_hosted?```, could we use it?

File: app/helpers/application_helper.rb [hunk 2]
```diff
@@ -229,6 +229,10 @@ def cartodb_com_hosted?
     Cartodb.config[:cartodb_com_hosted].nil? || Cartodb.config[:cartodb_com_hosted]
   end
 
+  def cartodb_onpremise_version
+    Cartodb.config[:onpremise_version]
+  end
+
```
Comment (xavijam) [inline app/helpers/application_helper.rb]: Ah! new flag... Should we check that if it is present and then "true"?

File: app/views/admin/organization_users/_form.html.erb [hunk 3]
```diff
@@ -100,7 +100,13 @@
...
       </div>
       <div class="Form-rowInfo OrganizationUser-quotaInfo js-quotaInfo">
-        <span class="js-assignedSize"><%= number_to_human_size(@user.quota_in_bytes) %></span> assigned, <button class="OrganizationUser-quotaInfoLink js-changeQuota">change</button> or <a href="<%= current_user.upgrade_url(request.protocol) %>">upgrade your account</a> if needed
+        <span class="js-assignedSize"><%= number_to_human_size(@user.quota_in_bytes) %></span> assigned, <button class="OrganizationUser-quotaInfoLink js-changeQuota">change</button> 
+        <% unless cartodb_com_hosted? %>
+        or <a href="<%= current_user.upgrade_url(request.protocol) %>">upgrade your account</a> if needed
+        <% else %>
+        it
+        <% end %>
+
```
Comment (xavijam) [inline app/views/admin/organization_users/_form.html.erb]: Good one, thanks.

File: app/views/admin/shared/_public_header.erb [hunk 4]
```diff
@@ -16,6 +16,7 @@
     </div>
     <div class="Header-settings">
       <ul class="Header-settingsList">
+        <% unless cartodb_com_hosted? %>
```
Comment (xavijam) [inline app/views/admin/shared/_public_header.erb]: @saleiva did some changes in the public header further to onpremise things, didn't you @saleiva?

File: lib/assets/javascripts/cartodb/fonts/lato_loader.js [hunk 5]
```diff
@@ -1,4 +1,4 @@
-var customInstall = cdb.config.get('cartodb_com_hosted');
+var customInstall = window.config.cartodb_com_hosted;
```
Comment (xavijam) [inline lib/assets/javascripts/cartodb/fonts/lato_loader.js]: Argggg, thanks for fixing it.

**Comment by xavijam** [comment]
Just those comments, but looks ok :+1:

File: app/helpers/application_helper.rb (hunk 2 above)
Comment (javisantana) [inline app/helpers/application_helper.rb]: it's not a flag, it's a string

Comment (xavijam) [inline app/helpers/application_helper.rb]: ahhh ok

File: app/views/admin/shared/_public_header.erb (hunk 4 above)
Comment (javisantana) [inline app/views/admin/shared/_public_header.erb]: what changes? where is the design?

**Comment by santisaez** [comment]
:+1:

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/3587/))

**Pull request merged**